docs/migration-guide.md

```diff
@@ -16,46 +16,54 @@ To ensure AWS your account is set up correctly, run:
 
 Then follow the instructions for any errors.
 
-## Initial setup
-Create a new GuCDK project in your repository following [this guide](setting-up-a-gucdk-project.md).
-This will create a new CDK (Typescript) app that simply wraps your existing Cloudformation template.
+## Process
+Generally speaking, we've found this to be a good process to follow when migrating a stack to GuCDK (assumes use of `npm`):
 
-Pay attention to the output of the command as it may describe further manual steps at this point.
-Once done, we recommend running `git diff` to see the full set of changes.
+### Introduce CDK to the repository with CI/CD
+1. Create a [new project](setting-up-a-gucdk-project.md) specifying the `--yaml-template-location` flag.
+2. Check to see what changes will be made to the stack:
 
-Finally, confirm that your new CDK stack is not materially different from your existing stack by running a diff:
+   ```shell
+   npm run diff -- --profile <AWS PROFILE NAME> <STACK ID FROM bin/cdk.ts>
+   ```
 
-```sh
-npm run diff
-```
-
-You might find that the diff contains a lot of noise due to tags that are automatically added by `@guardian/cdk`.
-Tag changes will not cause disruption to your infrastructure.
-If desired, you can view a diff which excludes the tagging changes by temporarily [amending your stack's props](https://github.com/guardian/cdk-playground/blob/26af40f6432cb94b3e5ef34648d422a7a6ee6b33/cdk/bin/cdk.ts#L6).
+   This should only show differences in how resources are tagged; GuCDK will add various tags to _all_ resources in the stack.
+   If you find this initial diff too noisy, you could temporarily exclude these tags by amending your [stack's props](https://guardian.github.io/cdk/interfaces/constructs_core.GuStackProps.html#withoutTags),
+   specifically set `withoutTags` to `true`.
 
-Once you are happy with the diff, we recommend running CDK synth as part of your CI process.
-See [here](./setting-up-a-gucdk-project.md) for more detail on CI/CD.
+4. Configure CI and CD. See [here](setting-up-a-gucdk-project.md) for more detail.
+5. Raise a PR and merge.
 
-## Migrating resources
+### Migration to CDK
+We now have something like this:
 
-After initial setup, you have something like this:
-
-    CDK(cfn.yaml) -> cfn.yaml
+```
+CDK(cfn.yaml) -> cfn.json
+```
 
-That is, CDK is merely wrapping an existing template. This is useful as a
-starting point. For example, to integrate with CI/CD.
+That is, CDK is merely wrapping an existing template. The end goal is for CDK to do all the work:
 
-The end goal, though, is for CDK to do all of the work:
+```
+CDK -> cfn.json
+```
 
-    CDK -> cfn.yaml
+In the previous phase, we also created a [snapshot test](https://docs.aws.amazon.com/cdk/v2/guide/testing.html#testing_snapshot) (`cdk/lib/*.test.ts`),
```

Comment:
It's definitely worth mentioning this; I have frequently paired with developers who are not aware of this test (or its purpose) after running the `new` command